django/views/debug.py

```diff
@@ -235,6 +235,10 @@ def get_traceback_frame_variables(self, request, tb_frame):
 
 class ExceptionReporter:
     """Organize and coordinate reporting on exceptions."""
+
+    html_template_path = Path(CURRENT_DIR, 'templates', 'technical_500.html')
```

Comment:
`CURRENT_DIR` is already a `Path` object, so we could join this with the `/` operator.

```suggestion
    html_template_path = CURRENT_DIR / 'templates' / 'technical_500.html'
```

Reply:
Sure, I figured I'd just go with the minimal possible change.